### ISSUE TITLE
Official docker image

### ISSUE THREAD

**Issue by T-vK** [dustinkirkland/hollywood#33]
There are a few hollywood Docker images out there, but they are all either outdated (on the one hand because nobody is releasing a new one when a new version of hollywood comes out, but also because the repositories on Ubuntu are updated way too rarely. Not to mention that they are all buggy).
It would be nice to have a Dockerfile in this repository that could always create an image for the latest hollywood version and in addition to that it would be nice to always have the latest Docker image available on Docker Hub.

I can provide the appropriate Dockerfile if you're interested and guide you through the simple process of building/publishing the Docker image if necessary.

Support for Docker would be really nice to get better cross-platform support, especially for Windows.

**Comment by dustinkirkland** [comment]
Umkay.  I think probably @jessfraz image is about the most "official" one there is.  Whattayathink, Jess?

**Comment by sebix** [comment]
https://github.com/dustinkirkland/hollywood/commit/afd6b1bd1df59974d903ae536e6644e88decc932 added a note to the readme recommending the docker image `dustinkirkland/hollywood`.
But this image does not exist.

**Comment by dustinkirkland** [comment]
Good catch — the `dustinkirkland/hollywood` DockerHub image referenced in the README didn't exist. Fixed in 08ff384 to point to `cgr.dev/chainguard/hollywood`, which is already recommended at the top of the README and actively maintained by Chainguard.
